test_runner/src/main/kotlin/ftl/cli/firebase/test/CommonRunCommand.kt

```diff
@@ -199,4 +199,10 @@ abstract class CommonRunCommand {
                 "which don't support ansi codes, to avoid corrupted output use `single` or `verbose`."]
     )
     var outputStyle: String? = null
+
+    @CommandLine.Option(
+        names = ["--full-junit-result"],
```

Comment:
When you are adding new option you should update also example `flank.yml`, `flank.ios.yml` and `README.md`